site/content/_events/2024-03-26-lynchburg_maker_faire.md

```diff
@@ -0,0 +1,29 @@
+---
+layout: event
+title: "2024 Maker Faire Lynchburg"
+event_date: 2024-03-26
+event_time_start: 11:00
+event_time_end: 16:00
```

Comment:
we can tweak the front matter later, but we should probably model it on something like ical or gcal, while taking into account Meetup's api and our specific rendering needs.

to share components across several models, it helps if some of the field names are shared between models (or collections). so as much as possible, we should try to keep front matter similar for posts/events and other similar models, but it's not essential.

Reply:
That is a good point.  These aren't quite yet settled upon.  In the PR description I call out:

> I'm not super happy yet with the front matter of the events. I put in what felt reasonable, but I suspect we may want to figure out what works well enough for us. Consider these variables flexible until fully decided upon. Variables to consider are:
> * How to attach an image to an event
> * Identifying location
> * Start and end datetimes